### PR TITLE
Ayush/cp

### PR DESCRIPTION
**Pull request by ayushkamat**
added docstring and changed the token value so the build doesnt fail lmao

### CODE REVIEW

File: latch/auth/pkce.py [hunk 1]
```diff
@@ -87,7 +88,7 @@ def __enter__(self, *args):
     def __exit__(self, *args):
         ...
 
-    def construct_challenge(self) -> (str, str):
+    def construct_challenge(self) -> Tuple[str, str]:
```
Comment (kennyworkman) [inline latch/auth/pkce.py:91]: any reason for this

File: tests/test_cli.py [hunk 2]
```diff
@@ -30,5 +29,5 @@ def test_cp(test_account_jwt):
     with open("foo.txt", "w") as f:
         f.write("foobar")
 
-    _cmd = ["latch", "cp", "foo.txt", "/foo.txt"]
-    _run_and_verify(_cmd, "Successfully copied foo.txt to /foo.txt.")
+    _cmd = ["latch", "cp", "foo.txt", "latch:///foo.txt"]
+    _run_and_verify(_cmd, "Successfully copied foo.txt to latch:///foo.txt.")
```
Comment (kennyworkman) [inline tests/test_cli.py:33]: can you write tests for the opposite

File: latch/cli/main.py [hunk 3]
```diff
@@ -66,15 +66,15 @@ def init(pkg_name: str):
...
+@click.argument("source_file", nargs=1)
+@click.argument("destination_file", nargs=1)
+def cp(source_file: str, destination_file: str):
+    """Copy local files to LatchData and vice versa.
 
     Visit docs.latch.bio to learn more.
     """
-    _cp(local_file, remote_dest)
-    click.secho(f"Successfully copied {local_file} to {remote_dest}.", fg="green")
+    _cp(source_file, destination_file)
```
Comment (kennyworkman) [inline latch/cli/main.py:76]: `cp`

File: latch/services/cp.py [hunk 4]
```diff
@@ -98,3 +110,64 @@ def cp(local_file: str, remote_dest: str):
...
+            doesnt_exist/dir2/sample.fa, which has the same content as the
+            remote file. Note the nonexistent directory is folded into the
+            name of the copied file.
+    """
+    local_dest = Path(local_dest).resolve()
+    token = retrieve_or_login()
+    headers = {"Authorization": f"Bearer {token}"}
+    data = {"source_path": remote_source}
+
+    # todo(ayush): change it so we don't hardcode api endpoints
```
Comment (kennyworkman) [inline latch/services/cp.py:152]: ye, one of us will do this quite soon

**Review by kennyworkman — CHANGES_REQUESTED** [review]

File: latch/auth/pkce.py (hunk 1 above)
Comment (ayushkamat) [reply latch/auth/pkce.py:91]: not in particular - the linter that im using enforces super explicit type annotations so i changed it. i dont really have any strong opinions on this so i dont mind changing it back

File: tests/test_cli.py (hunk 2 above)
Comment (ayushkamat) [reply tests/test_cli.py:41]: done

**Pull request closed without merging**